src/wwwroot/js/site.js

```diff
@@ -509,19 +509,36 @@ function load_image_file(elem) {
         let reader = new FileReader();
         reader.addEventListener("load", () => {
             elem.dataset.filedata = reader.result;
-            preview.innerHTML = `<button class="interrupt-button auto-input-image-remove-button" title="Remove image">&times;</button><img alt="Image preview" />`;
-            let img = preview.querySelector('img');
-            img.onload = () => {
-                label.textContent = `${name} (${img.naturalWidth}x${img.naturalHeight}, ${describeAspectRatio(img.naturalWidth, img.naturalHeight)})`;
-                elem.dataset.width = img.naturalWidth;
-                elem.dataset.height = img.naturalHeight;
+            let mediaPreviewHtml;
```

Comment:
i think this is code specifically for image inputs? It should not be changed at all by an audio pr